### DISCUSSION TITLE
DDT4ALL plante lors de validation calculateur - win 7 32 bits

### DISCUSSION THREAD

**Discussion by Ludo25081985** [cedricp/ddt4all#933] [Q&A]
Bonjour,
Tous d'abord merci à  Cedric pour sont logiciel.
J'ai un souci lors de mon installation, je suis en windows 7 32 bits.
J'installe la version Python 3.8.6en x86, j'installe la base de données, tous ce passe bien, le logiciel trouve 2138 calculateur au démarrage.
J'ai fait un essai en mode non connecté, je valide véhicule,  TBD, le calculateur voulu , et des que je reclique dessus pour afficher les possibilités le logiciel plante sans que j'ai le temp de voir ce qui s'affiche sur l'autre page de commande.
J'ai essayé d'autre version , il y'a que celle ci qui marche, sur n'importe quelle autres calculateur sa fait pareil.
Qu'elle serais la cause de se plantage s'il vous plaît ?
Si un âme charitable pourrais m'aider ce serais sympa merci.

**Comment by AndyIreland** [comment]
Hello - try have a go and try and download this one and see if it crashes , its got the database built in , choose "All Cars" 
https://www.drive2.ru/b/498093336985338243/
- uninstall the one you have installed first.

**Comment by Ludo25081985** [comment]
Est ce que ça prend en compte le 32 bits , je ne pense pas si ??

**Comment by Ludo25081985** [comment]
Merci pour le retour déjà c'est cool

**Comment by Ludo25081985** [comment]
Je vais essayer quand même,  mais j'aurais préféré utilisé la version de cedric, je pense que c'est juste une modification à faire sur fichier écu.py mais je ne sais pas comment l'ouvrir ni l'écrire.

**Comment by Ludo25081985** [comment]
Ça marche avec le lien russe merci Andy,  mais si quelqu'un à la solution pour le logiciel originel de cedric je serais preneur.

**Comment by Furtif** [comment]
utilise le installateur prêt a l'emploi https://github.com/cedricp/ddt4all/releases/tag/v2.1.1 (et place la data base dans le dossier ou vous l'installais generalement c:\Program Files\DDT4ALL)

**Comment by Furtif** [comment]
Télécharge la version 32 bits pour win7 si vous avais win10 ou 11 architectures 64 bits télécharge la version x64

**Reply by Ludo25081985** [reply]
Bonjour , c'est exactement ce que j'ai fait de base mais sa me plante comme j'est expliquer plus haut

**Reply by Furtif** [reply]
Ok vais voir cela de pres donnez mói le véhicule et le calculateur pour que je puis faire quelques tests.

Merci

**Reply by Ludo25081985** [reply]
Merci beaucoup à toi.
C'est un Trafic 3 , calculateur de tdb 
Mais sa le.fait sur tous les calculateurs.
Ça le charge en 1 milliseconde et plante , pas le.temp de voir.
Je suis en win 7 32 bits, et j'ai mis les dossier de base de données dans programmes / ddt4all,  peut être faut il les mettre dans programmes/ddt4all/python38 ??

**Reply by Furtif** [reply]
![image](https://github.com/cedricp/ddt4all/assets/11375035/2acb0d91-9639-4720-8b73-0fdcdcbac5d9)
bon chelou votre erreur!!! 

je pense qu'il vous manque peut-etre un dll dependencies 
Installez [Python 3.8.6 Windows x86 executable installer](https://www.python.org/ftp/python/3.8.6/python-3.8.6.exe)
ou 
[Python 3.8.6 Windows x86 web-based installer](https://www.python.org/ftp/python/3.8.6/python-3.8.6-webinstall.exe) 

[python 3.8.6](https://www.python.org/downloads/release/python-386)

puis testez pour voir.

**Reply by Furtif** [reply]
si problème persiste supprimez tout reinstallez e utilisez cette DB copiez le zip ecu dans le dossier de ddt4all https://github.com/cedricp/ddt4all/discussions/845

**Comment by KarelSvo** [comment]
@Ludo25081985 

https://github.com/cedricp/ddt4all/discussions/930#discussioncomment-7490140

**Reply by Ludo25081985** [reply]
Merci je testerai ça quand j'aurais le temp et vous mettrais une capture de la fenêtre de commande.

**Comment by Furtif** [comment]
installer win7 32 bits dans un hp très vieux fonctionne parfaitement avec la data base plus haut donné et sans faire rien d'autre.
